src/borg/testsuite/archiver/create_cmd.py

```diff
@@ -169,14 +168,13 @@ def test_create_duplicate_root(archivers, request):
 
 
 @pytest.mark.skipif(is_win32, reason="unix sockets not available on windows")
-def test_unix_socket(archivers, request):
+def test_unix_socket(archivers, request, tmp_path):
     archiver = request.getfixturevalue(archivers)
     repo_location = archiver.repository_location
     cmd(archiver, f"--repo={repo_location}", "rcreate", RK_ENCRYPTION)
     try:
-        name = tempfile.mktemp(prefix="test_sock", suffix=".sock")  # short path needed for socket.bind
         sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
-        sock.bind(name)
+        sock.bind(os.path.join(archiver.input_path, "unix-socket"))
```

Comment:
I suspect that this change breaks the tests somewhere. That "short path needed for socket.bind" was there for a reason and the paths created by pytest are rather longer.